examples/error-handling/error_handling.out

```diff
@@ -1,4 +1,4 @@
 # To run this sample, navigate to the directory that contains the
 # `.bal` file and issue the `ballerina run` command.
 $ ballerina run error_handling.bal
-Error occurred: Account with ID: 24 is not found, accountID: 24
+Error: Account Not Found, Account ID: -1
```

Comment:
```suggestion
Error: Account ID: -1 not found.
```

Reply:
this is part of the example itself.

error reason: Account Not Found
error details contains ID info

Reply:
+1